pages/docs/data-ops/continuous-integration/github-actions.mdx

```diff
@@ -168,6 +168,54 @@ jobs:
         run: exit "${{ steps.create-hb-build.outputs.exitCode }}"```
 ````
 
+### Deploying dbt Core + Hashboard to production
+
+This action runs when commits are merged to your `main` branch.  It runs dbt against the production target and then generates and applies a Hashboard build.
+
+````yml
+name: Deploy DBT and Hashboard
+
+on:
+  push:
+    branches: [main]
+
+env:
+  BQ_DATASET_NAME: "ci_pr_${{ github.event.pull_request.number }}"
+  BQ_DATASET_DESCRIPTION: "${{ github.event.pull_request.title}} (${{github.event.pull_request.html_url}})"
+jobs:
+  dbt-and-hashboard-preview:
+    name: "Run dbt pipeline and create Hashboard preview"
+    concurrency:
+      group: preview-${{ github.event.pull_request.number || github.ref }}
+      cancel-in-progress: true
+    runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.merged }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Install dependencies"
+        run: |
+          pip install dbt-core dbt-bigquery
+          dbt deps
+          dbt --version
+
+      - name: dbt production build
+        id: dbt-build
+        run: |
+          dbt build --target=production
+
+      - name: Create and deploy Hashboard build
+        id: create-hb-build
+        env:
+          HASHBOARD_PROJECT_ID: ${{ secrets.HASHBOARD_PROJECT_ID }}
+          HASHBOARD_ACCESS_KEY_ID: ${{ secrets.HASHBOARD_ACCESS_KEY_ID }}
+          HASHBOARD_SECRET_ACCESS_KEY_TOKEN: ${{ secrets.HASHBOARD_SECRET_ACCESS_KEY_TOKEN }}
+          HB_DBT_USE_DEFER: false
+          HB_DBT_INCLUDE_DB_IN_SCHEMA: false
+        run: |
+          hb build --dbt-artifacts=./target && hb build apply --no-confirm
```

Comment:
My bad about the note on slack, that doesn't apply when passing build artifacts directly - when we pass the target folder we don't run any dbt commands (including the schema qualification step) so the env vars are actually not necessary in this case

Reply:
Ah ok perfect